genapkovl-lima.sh

```diff
@@ -223,10 +225,15 @@ fi
 
 if [ "${LIMA_INSTALL_CNI_PLUGIN_FLANNEL}" == "true" ]; then
     echo "cni-plugin-flannel" >> "$tmp"/etc/apk/world
-    ARCH=amd64
+    if [ "$(uname -m)" == "x86_64" ]; then
+        ARCH=amd64
+    fi
     if [ "$(uname -m)" == "aarch64" ]; then
         ARCH=arm64
     fi
+    if [ "$(uname -m)" == "armv7l" ]; then
+        ARCH=armv7
+    fi
```

Comment:
Why do we set `ARCH` here at all? It isn't being used anymore at all. It seems to be vestigial code from the time when this block ended with:

```
    mkdir -p "${tmp}/usr/libexec/cni"
    ln -s "flannel-${ARCH}" "${tmp}/usr/libexec/cni/flannel"
```

But that has since been removed because the `cni-plugin-flannel` package has since been fixed. So we can drop all the `ARCH` setting here.